src/mocks/handlers.js

```diff
@@ -2,10 +2,6 @@ import { rest } from 'msw';
 
 export const handlers = [
   rest.get('/https://pokeapi.co/api/v2/pokemon/', (req, res, ctx) => {
-    return res(
-      ctx.json({
-        welcomeMessage: 'Hello World!',
-      })
-    );
+    return res(ctx.json(res.data));
```

Comment:
Instead of returning the actual data just copy and paste a valid response. I think what you are doing here is still calling the actual endpoint, which is what we want to avoid.